newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -672,6 +727,20 @@ export default class AuthenticatedUserProvider extends React.Component<
     });
   };
 
+  openAdditionalUserInfoDialog = (open: boolean = true) => {
```

Comment:
Is the default value useful?

Reply:
I copied other functions in this file to be consistent.
It allows calling this function without passing a param